frontend/src/pages/LoginPage.js

```diff
@@ -20,8 +22,35 @@ export default function LoginPage() {
     }
   }, [ email, password ]);
 
+  // payload for post a new user
+  const payload = {
+    email: email,
+    password: password,
+  };
+
+  // function to log in a user
   function loginUser(e) {
     e.preventDefault();
+
+    UserAuthApi.login(payload).then(res => res.json()).then(data => {
+      localStorage.setItem('token', data.token);
+
+      if (data.token !== undefined || data.token !== null) {
+        Swal.fire(
+          'Congratulation!',
+          'You have successfully logged in to E-Learning System.',
+          'success'
+        ).then(() => {
+          window.location.replace('http://localhost:3000/dashboard');
```

Comment:
instead of using `window.location.replace` react-router-dom have `history.push()` function that you may use to redirect user to the dashboard page

Reply:
Thank you Sir. I'll replace this one.